src/style.css

```diff
@@ -1 +1,26 @@
 /* add styles here */
+body {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). It's better to style elements using `.classes`. Consider adding a class to the elements with the `.stars` class and style them accordingly.